.github/workflows/lint.yml

```diff
@@ -0,0 +1,60 @@
+name: Lint Obelix
+
+on:
+  push:
+    branches: ["master", "develop"]
+  pull_request:
+    branches: ["master", "develop"]
+
+defaults:
+  run:
+    shell: bash
+
+jobs:
+  linter:
+    name: Run linter in Idefix and Panoramix
```

Comment:
on?

Reply:
Nice catch, fixed in https://github.com/Cleopha/ifttt-like/pull/30/commits/91e3d7ee7d3b21bf95a5f309f9c27414aa071348